packages/onboarding-ui/src/pages/LoginPage/LoginPasswordLessForm.tsx

```diff
@@ -0,0 +1,69 @@
+import { FieldGroup, Field, TextInput, Button } from '@rocket.chat/fuselage';
```

Comment:
`form` files should be inside of `forms` folder and I'm missing the tests for them

Reply:
Additionally there's a bigger space between the fields and the button in the `form`!